src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -325,6 +326,27 @@ public void update(long time) {
             packet.process(this);
         }
 
+        //System.out.println(getAcquiredElement().getHandler().getBatchThread());
```

Comment:
Should this block be removed?

Reply:
Will be once the PR is ready, for now it is pretty useful to test performance